doc/articles/controls/CommandBar.md

```diff
@@ -149,8 +149,10 @@ Gets or sets a brush that describes the foreground color.
 
 #### Remarks
 
-* This is typically used to change the `Content`'s text color.
+* This is typically used to change the Content`'s text color.
 * Only supports `SolidColorBrush`.
+* Setting this property will not affect any of the `CommandBar's` `AppBarButton` tint Color. If you need to change the `AppBarButton` tint, this is possible by setting the `ShowAsMonochrome` property to true as well as setting the Foreground`SolidColorBrush`on the`BitmapIcon`.
```

Comment:
You should mention the `FeatureConfiguration.AppBarButton.EnableBitmapIconTint` property here